examples/combined_ODNP.py

```diff
@@ -0,0 +1,333 @@
+from numpy import *
+from pyspecdata import *
+import os
+import sys
+import SpinCore_pp
+import time
+from Instruments import power_control
+from datetime import datetime
+from SpinCore_pp.verifyParams import verifyParams
+from SpinCore_pp.power_helper import gen_powerlist
+from pyspecdata.file_saving.hdf_save_dict_to_group import hdf_save_dict_to_group
+import h5py
+# {{{ from run_Hahn_echo_mw.py
+fl = figlist_var()
+save_file=True
+# {{{ experimental parameters
+# {{{ these need to change for each sample
```

Comment:
note this comment -- some of my conversations below refer to the fact that this block at the top should be reserved for things that change every time.